packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -109,6 +117,10 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
             `Validate ${models.length} models in manifest with version ${manifestVersion}`,
         );
 
+        if (models.length === 0) {
+            throw new ParseError(`No models found`);
```

Comment:
ParseError or NotFoundError ?

Reply:
Parse error because it relates to the arguments rather than database results.